css/properties.json

```diff
@@ -1720,6 +1720,22 @@
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
+  "alignment-baseline": {
+    "syntax": "baseline | alphabetic | ideographic | middle | central | mathematical | auto | before-edge | after-edge | text-before-edge | text-after-edge | hanging",
+    "media": "none",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Inline"
+    ],
+    "initial": "baseline",
+    "appliesto": "inlineLevelBoxesFlexItemsGridItemsTableCellsAndSVGTextContentElements",
```

Comment:
I don't see this value defined in css/properties.schema.json. Could you mention the source of your info?

Reply:
Okay, I see that you're adding an entry for this in the schema file